src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Program.cs

```diff
@@ -14,19 +13,28 @@ namespace Microsoft.Health.Fhir.Liquid.Converter.Tool
 {
     public class Program
     {
-        public static async Task Main(string[] args)
+        public static async Task<int> Main(string[] args)
         {
             var parseResult = Parser.Default.ParseArguments<ConverterOptions, PullTemplateOptions, PushTemplateOptions>(args);
-            parseResult.WithParsed<ConverterOptions>(options => ConverterLogicHandler.Convert(options));
-            await parseResult.WithParsedAsync<PullTemplateOptions>(options => TemplateManagementLogicHandler.PullAsync(options));
-            await parseResult.WithParsedAsync<PushTemplateOptions>(options => TemplateManagementLogicHandler.PushAsync(options));
-            parseResult.WithNotParsed((errors) => HandleOptionsParseError(parseResult));
+            try
+            {
+                parseResult.WithParsed<ConverterOptions>(options => ConverterLogicHandler.Convert(options));
+                await parseResult.WithParsedAsync<PullTemplateOptions>(options => TemplateManagementLogicHandler.PullAsync(options));
+                await parseResult.WithParsedAsync<PushTemplateOptions>(options => TemplateManagementLogicHandler.PushAsync(options));
+                parseResult.WithNotParsed((errors) => HandleOptionsParseError(parseResult));
+                return 0;
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine($"Process failed: {ex.Message}");
+                return -1;
```

Comment:
return 1?

Reply:
@yankunhuang-pku What about return 1 when process failed?

Reply:
As long as it is a non-zero number, I can identify it on my side.